astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -42,24 +40,24 @@ def assert_table_equal(t1, t2, check_meta=False, rtol=1.0e-15, atol=1.0e-300):
     Test equality of all columns in a table, with stricter tolerances for
     float columns than the np.allclose default.
     """
-    assert_equal(len(t1), len(t2))
-    assert_equal(t1.colnames, t2.colnames)
+    npt.assert_equal(len(t1), len(t2))
+    npt.assert_equal(t1.colnames, t2.colnames)
     if check_meta:
-        assert_equal(t1.meta, t2.meta)
+        npt.assert_equal(t1.meta, t2.meta)
     for name in t1.colnames:
         if len(t1) != 0:
-            assert_equal(t1[name].dtype.kind, t2[name].dtype.kind)
+            npt.assert_equal(t1[name].dtype.kind, t2[name].dtype.kind)
         if not isinstance(t1[name], MaskedColumn):
             for i, el in enumerate(t1[name]):
                 try:
                     if not isinstance(el, str) and np.isnan(el):
-                        assert_true(
+                        npt.assert_(
                             not isinstance(t2[name][i], str) and np.isnan(t2[name][i])
                         )
```

Comment:
```suggestion
                        assert not isinstance(t2[name][i], str) and np.isnan(t2[name][i])
```
See discussion; I don't think we want to use `npt.assert_` at all after that.